src/webhooks/webhooks.service.ts

```diff
@@ -67,4 +83,56 @@ export class WebhooksService {
       return error;
     }
   }
+
+  async upsertStory({ action, story_id }: StoryDto) {
```

Comment:
typo on upsert -> update

Reply:
I used upsert on purpose. Insert if it doesn't exist or update if it does

Reply:
ah lol gotcha!